pandas/compat/pickle_compat.py

```diff
@@ -105,22 +105,8 @@ def __new__(cls) -> "DataFrame":  # type: ignore
 # If classes are moved, provide compat here.
 _class_locations_map = {
     ("pandas.core.sparse.array", "SparseArray"): ("pandas.core.arrays", "SparseArray"),
-    # 15477
-    #
-    # TODO: When FrozenNDArray is removed, add
```

Comment:
Kudos to @gfyoung for making this dead simple

Reply:
@WillAyd : It's only simple if it was still correct 🙂.  A lot could have happened in a year since #23396.